docs/sources/reference/components/pyroscope/pyroscope.ebpf.md

```diff
@@ -121,7 +127,7 @@ Symbols are extracted from various sources, including:
 - The `.symtab` and `.dynsym` sections in the debug ELF file.
 - The `.gopclntab` section in Go language ELF files.
 
-The search for debug files follows [gdb algorithm](https://sourceware.org/gdb/onlinedocs/gdb/Separate-Debug-Files.html).
+The search for debug files follows [gdb algorithm].
```

Comment:
```suggestion
The search for debug files follows [gdb algorithm][].
```

We can add the extra `[]` just for consistency. It's not required but we tend to use it throughout the rest of the docs.